client/src/utils/create-sendgrid-contact.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+// Sendgrid secrets
+const sendgrid_contacts_url = process.env.REACT_APP_SENDGRID_CONTACS_API_URL;
+const sendgri_api_key = process.env.REACT_APP_SENDGRID_API_KEY;
+const sendgrid_list_id = process.env.REACT_APP_SENDGRID_LIST_ID;
+
+let sendgridContactsObject = {
+  contacts: [
+    {
+      list_ids: [sendgrid_list_id],
+    },
+  ],
+  list_ids: [sendgrid_list_id],
+};
```

Comment:
This is because SendGrid expects this object, specifically on the request body, if it is not exactly like this, then the req will fail. Also, the documentation has the wrong URL, which was annoying and thought it was issues with Auth but Nah, they just don't have really good documentation...